src/aap_eda/services/ruleset/activate_rulesets.py

```diff
@@ -402,60 +413,63 @@ def activate_in_k8s(
         activation_id = activation.pk
         job_name = f"activation-job-{activation_id}-{activation_instance.id}"
         pod_name = f"activation-pod-{activation_id}-{activation_instance.id}"
+        secret_name = None
+        job_spec = None
+
+        if not reconnect:
```

Comment:
So `reconnect` logic here expects the k8s container, secret, service, etc. are all available when reconnection happens, right?

Reply:
Yes. Here it skips creating container/secret/service etc. Later when it actually reconnects the container, it will check whether the container exists.